test/testcafe/spec/EnrollProfileView_spec.js

```diff
@@ -128,8 +128,10 @@ test.requestHooks(requestLogger, EnrollProfileSignUpWithBooleanFieldsMock)('shou
 
   await t.expect(await enrollProfilePage.clickRadioButton('userProfile.pet', 0)).eql('Yes');
 
-  await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.subscribe')).eql(false);
-  await enrollProfilePage.setCheckbox('userProfile.subscribe');
+  await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.required')).eql(false);
+  await t.expect(await enrollProfilePage.getCheckboxValue('userProfile.optional')).eql(false);
+  await enrollProfilePage.setCheckbox('userProfile.required');
+  await enrollProfilePage.setCheckbox('userProfile.optional');
```

Comment:
After checking the checkboxes can we also make sure the checkboxValue is set to `true`?

Reply:
I think we had a discussion before. from ui, after set the checkbox is selected, but not sure why equal to `true` do not work...

Reply:
resolved offline